providers/capvcd/upgrade/test_data/cluster_values.yaml

```diff
@@ -5,7 +5,15 @@ controlPlane:
   # https://github.com/giantswarm/giantswarm/issues/29353
   replicas: 1
   sizingPolicy: m1.large
-  template: ubuntu-2004-kube-v1.25.13
+  template: flatcar-stable-3602.2.1-kube-v1.25.16
+  dns:
+    imageRepository: gsoci.azurecr.io/giantswarm
+    imageTag: 1.9.4-giantswarm
```

Comment:
Isn't this already the default? https://github.com/giantswarm/cluster-cloud-director/blob/main/helm/cluster-cloud-director/values.yaml#L28

Reply:
I forgot I added it to the default values

Reply:
We can also leave etcd as default as it is the repo that mattered.